includes/views.inc

```diff
@@ -9,7 +9,13 @@
  * Implements hook_preprocess_HOOK().
  */
 function gesso_preprocess_views_view(array &$variables) {
-  $variables['path'] = $variables['view']->getRequest()->getPathInfo();
+  /** @var \Drupal\views\ViewExecutable $view */
+  $view = $variables['view'];
+
+  $variables['path'] = $view->getRequest()->getPathInfo();
+  $variables['view_title'] = [
```

Comment:
Two things here: 1) I think this should be `$variables['title']` instead of `$variables['view_title']` so it matches Drupal's core `views-view.html.twig` file. This also makes the mapping in the template file optional (although I generally prefer explicit mappings anyway so it's clear what variables are expected). 2) I think it would be good to add a check here first so that we're only getting the title if we don't already have it, either by checking `$variables['view']` or by checking the view display mode.